packages/adapter-node/tests/smoke.js

```diff
@@ -0,0 +1,33 @@
+import { test } from 'uvu';
+import { createServer } from '../src/server.js';
+import * as assert from 'uvu/assert';
+import fetch from 'node-fetch';
+
+const { PORT = 3000 } = process.env;
+
+function startServer() {
+	const server = createServer({ render: () => {} });
+	return new Promise((res, rej) => {
```

Comment:
Everywhere else in the repo we use `fulfil` and `reject` (I've always found 'resolve' ambiguous, and generally advocate for avoiding unnecessary abbreviations)

```suggestion
	return new Promise((fulfil, reject) => {
```

Reply:
By contrast, `fulfil` is even more specific and precise, because it means '`fulfill`, except Rich wrote it'.